### DISCUSSION TITLE
Option to export album as txt file showing the file path of all items in the album

### DISCUSSION THREAD

**Discussion by thisisme78** [photoprism/photoprism#4460] [Ideas]
Hi

I think it would be super helpful to allow exporting an album as a text file listing the original file name (including file path) of each item in the library.

Use case example : let say I want to fix the GPS coordinate exif metadata of a bunch of photos
1. create an album with all the photos I want to fix
2. export the album as text and gets a list /filepath_1/photo_n
...
/filepath_1/photo_n
3. run a small script that does what I want, looping on the file
4. Reindex the folders
5. Enjoy having the photos at the corrected gps position

The file can also be used to open the files in an external program, etc.

Extra feature: offer to reindex items in the album

**Comment by graciousgrey** [comment]
Thank you for your feedback!

We do have a somewhat similar feature request here: https://github.com/photoprism/photoprism/issues/1518.

**Comment by thisisme78** [comment]
Indeed, 1518 is close enough. I see it is in the roadmap. It would be a really great feature. No software should aim at doing everything I guess and 4460 or 1518 would allow to build bridges with other tools.
